docs/4.3. Behaviour - Tokens.md

```diff
@@ -44,19 +41,16 @@ The `sub` (subject) claim MUST be included in the token. This claim MUST contain
 end-user by the user authorization system. For example, this may be a username in a Single Sign-On (SSO) system, or the
 email address of the user.
 
-For the authorization code grant, the subject typically identifies the authorized accessor for which the access token is
-being requested.
+For the authorization code grant, the subject typically identifies the authorized accessor for which the access token
+is being requested.
 
 #### aud
 _Identifies the recipients of the JWT_
 
-The `aud` (audience) claim MAY be included in the token, but MUST be included where an AMWA NMOS Specification with
-which the token is to be used requires it.
-
-AMWA NMOS APIs are free to specify their own audience claims, but in the absence of specification resource servers
-SHOULD assume that the audience claim contains the fully resolved domain name of the intended recipient, or a domain
-name containing wild-card characters in order to target a subset of devices on a network. Such wild-carding of domain
-names is documented in [RFC 4592][RFC-4592].
+The `aud` (audience) claim MUST be included in the token. AMWA NMOS APIs are free to specify their own audience
```

Comment:
"AMWA NMOS APIs are free to specify their own audience claims"

I'm not really sure what this means. Unless we have a good idea of a future use case I'd suggest we simply state that the 'aud' claim is a domain name which may include wildcards as specified.